emulators/mame/files/mame-patch-0226-scripts-genie-compile-warnings.diff

```diff
@@ -1,6 +1,13 @@
---- scripts/genie.lua	2020-11-28 16:02:27.000000000 -0500
-+++ scripts/genie.lua	2020-11-28 17:01:38.000000000 -0500
-@@ -1002,6 +1002,7 @@
+--- scripts/genie.lua	2020-12-11 20:39:27.000000000 -0500
++++ scripts/genie.lua	2020-12-11 20:39:15.000000000 -0500
+@@ -996,12 +996,13 @@
+ 	buildoptions {
+ 		"-Wall",
+ 		"-Wcast-align",
+-		"-Wundef",
++		"-Wno-undef",
```

Comment:
all the rest of MacOS and MacPorts are moving in the opposite direction, to ADD warnings and errors for undefined functions rather than suppress them... apparently for arm64 this is critical. Are we really sure that in this one case, we need to do the opposite?

Reply:
Good to know, as I wasn't aware. These can certainly be turned back on when ARM64 is tackled... or that can be done now.

Thoughts?

Reply:
This has been added to the ARM64-related ticket: [61795](https://trac.macports.org/ticket/61795)